spyder/app/tests/test_mainwindow.py

```diff
@@ -105,6 +126,76 @@ def close_window():
 # Tests
 #==============================================================================
```

Comment:
Add here (i.e. before the test) something like:

    # IMPORTANT NOTE: Please leave this test to be the first one here to
    # avoid possible timeouts in Appyevor